libs/api-test-environment/src/main/java/de/cyface/apitestutils/fixture/TestMeasurementDocument.java

```diff
@@ -93,7 +93,7 @@ public Future<String> insert(final MongoClient mongoClient) {
                 .put(METADATA_OS_VERSION_FIELD, "Android 9.0.0")
                 .put(METADATA_APP_VERSION_FIELD, "1.2.0")
                 .put(METADATA_LENGTH_FIELD, 1500.2)
-                .put(USER_ID_FIELD, ownerUserId)
+                .put(USER_ID_FIELD, new JsonObject().put("$oid", ownerUserId)) // new ObjectId() not supported
```

Comment:
Where I added those comments I tried to store `new ObjectId(userIdString)` but this did lead to an exception.